src/pages/ReservationDetails.tsx

```diff
@@ -13,146 +26,259 @@ export default function ReservationApproval() {
+      });
+  };
 
   return (
-    <div className='booking-summary-container'>
-      <h3>Reservation summary</h3>
-      <div className='booking-summary-date_person-container'>
-        <div>
-          <p className='booking-summary-label'>Date</p>
-          <DatePicker
-            value={reservation.time?.from ?? 0}
-            onChange={() => {
-              return;
-            }}
-            renderInput={params => (
-              <TextField {...params} size='small' fullWidth disabled />
+    <>
+      {error ? (
+        error !== '' ? (
+          <Alert severity='error' style={{ margin: '30px' }}>
+            <AlertTitle>{error}</AlertTitle>
+            Please try again later
+          </Alert>
+        ) : (
+          <Alert severity='error' style={{ margin: '30px' }}>
+            <AlertTitle>Holy Guacamole!</AlertTitle>
+            Something went wrong —{' '}
+            <strong>please contact the server administrator</strong>
+          </Alert>
+        )
+      ) : null}
+      <div className='booking-summary-container'>
+        <div className='booking-summary-header'>
+          <h3>Reservation summary</h3>
+          {reservation ? (
+            <Tooltip title='24 hours before your reservation you can confirm it!'>
+              {reservation.confirmed ? (
+                <Chip icon={<CheckIcon />} label='Confirmed' color='success' />
+              ) : (
+                <Chip
+                  icon={<CloseIcon />}
+                  label='Not Confirmed'
+                  color='error'
+                />
+              )}
+            </Tooltip>
+          ) : null}
+        </div>
+        <div className='booking-summary-date_person-container'>
+          <div>
+            <p className='booking-summary-label'>Date</p>
+            {reservation ? (
+              <DatePicker
+                value={reservation?.time?.from ?? null}
+                onChange={() => {
+                  return;
+                }}
+                renderInput={params => (
+                  <TextField {...params} size='small' fullWidth disabled />
+                )}
+                disabled={true}
+              />
+            ) : (
+              <Skeleton variant='rectangular' height={25} />
             )}
-          />
+          </div>
+          {/* TODO */}
+          <div>
+            <p className='booking-summary-label'># of persons</p>
+            {reservation ? (
+              <TextField
+                id='outlined-number'
+                type='number'
+                defaultValue={2}
+                InputProps={{
+                  readOnly: true,
+                }}
+                size='small'
+                disabled
+              />
+            ) : (
+              <Skeleton variant='rectangular' height={25} />
+            )}
+          </div>
         </div>
 
-        <div>
-          <p className='booking-summary-label'># of persons</p>
+        <p className='booking-summary-label'>Time</p>
```

Comment:
Wäre nice, hier from und to zu haben, statt nur "Time":
![image](https://user-images.githubusercontent.com/30153207/177870866-5c1eb201-4e97-4993-a0dd-bdb70fc55edc.png)